src/Core/Framework/Adapter/Filesystem/FilesystemFactory.php

```diff
@@ -137,7 +137,9 @@ private function resolveFilesystemConfig(array $config): array
 
     private function getFallbackUrl(): string
     {
-        $request = Request::createFromGlobals();
+        // Change from use Request::createFromGlobals because files in $_FILES could be deleted
+        $request = new Request($_GET, [], [], [], [], $_SERVER);
```

Comment:
maybe instead call `$request->files->replace();` after `$request = Request::createFromGlobals();`

I am not sure, how important post cookies and stuff are

Reply:
That won't work, because at this point the exception is already thrown. If you create the request, it  also creates UploadedFile objects from the $_FILES var. 

For a fallback URL you don't need Cookies, Files or Post data :shrug:

Reply:
okay :+1: 

to have this looking nicer, you may want to use named parameter